api4/command_test.go

```diff
@@ -627,7 +627,7 @@ func TestExecuteCommandAgainstChannelOnAnotherTeam(t *testing.T) {
 	}()
 	th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableCommands = true })
 	th.App.UpdateConfig(func(cfg *model.Config) {
-		*cfg.ServiceSettings.AllowedUntrustedInternalConnections = "localhost 127.0.0.1"
+		*cfg.ServiceSettings.AllowedUntrustedInternalConnections = "localhost,127.0.0.1"
```

Comment:
Curious, were the tests failing if you didn't update these or was this just to make sure your changes worked?

Reply:
They worked before and after (without mods) since the PR is backward compatible. I made the changes only to keep consistency with other admin settings as the original ticket suggested as I understood we wanted the comma separated to be the "default" way, which reminds me that if this is indeed what we want we should probably update the part in the docs where this setting is documented.

Reply:
Makes sense 👍 Could we add some tests to make sure that it works using both the old way (space delimited) and the new way (comma delimited)?

Reply:
Indeed. Just added some tests to make sure the splitting is backward compatible and properly handles all required cases.